src/components/Pages/MainPage/MainPage.tsx

```diff
@@ -76,19 +133,23 @@ function MainPage(props: Props) {
           <Divider className={styles.divider} />
           {!isBlocked ? (
             <>
-              <Tabs>
-                {tabs.map(tab => {
-                  return (
-                    <Tabs.Tab key={tab.value} active={selectedTab === tab.value} onClick={() => handleTabChange(tab.value)}>
-                      <span className={styles.tab}>{tab.displayValue}</span>
-                    </Tabs.Tab>
-                  )
-                })}
-              </Tabs>
+              <div ref={tabsRef}>
+                <Tabs>
+                  {tabs.map(tab => {
+                    return (
+                      <Tabs.Tab key={tab.value} active={selectedTab === tab.value} onClick={() => handleTabChange(tab.value)}>
+                        <span className={styles.tab}>{tab.displayValue}</span>
+                      </Tabs.Tab>
+                    )
+                  })}
+                </Tabs>
+              </div>
               {selectedTab === AccountTabs.ASSETS ? (
                 <Assets profileAddress={profileAddress ?? nullAddress} view={view} />
               ) : selectedTab === AccountTabs.CREATIONS ? (
                 <Creations profileAddress={profileAddress ?? nullAddress} view={view} />
+              ) : selectedTab === AccountTabs.REFERRAL && profileAddress ? (
+                <Referrals view={view} profileAddress={profileAddress} />
```

Comment:
It seems that the `view` prop is not used in the `Referrals` component, maybe there's something missing?
```suggestion
                <Referrals profileAddress={profileAddress} />
```